django_prices_vatlayer/utils.py

```diff
@@ -57,29 +62,47 @@ def create_objects_from_json(json_data):
     for code, value in rates.items():
         VAT.objects.update_or_create(
             country_code=code, defaults={'data': value})
+        get_tax_rates_for_country(code, force_refresh=True)
```

Comment:
Wouldn't it be easier if `update_or_create` returned a structure that could be passed directly to `cache.set`?